src/css/icons.css

```diff
@@ -1,10 +1,3 @@
-@import '../../node_modules/file-icons-js/css/style.css';
-
 .database-icon::before {
   left: 2px;
 }
-
-.dark {
```

Comment:
this will be a breaking change 🤔 95% of the colors in the light mode and dark mode are the same in `icondb.js`.